CONTRIBUTING.md

```diff
@@ -2,20 +2,74 @@
 
 We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
 
-The easiest way to reach us is on the [Kubernetes slack #kyverno channel](https://app.slack.com/client/T09NY5SBT/CLGR9BJU9). 
 
-## Ways you can contribute
-- Report potential bugs
-- Request a feature
-- Request a sample policy
-- Join our community meetings
-- Submit a PR for [open issues](https://github.com/kyverno/kyverno/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-- Fix or improve documentation
+## Ways you can contribute:
+**This page contains:**
+   - [Report Issues](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#report-issues)
+   - [Submit Pull Requests](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#submit-pull-requests)
+   - [Fix or improve documentation](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md#fix-or-improve-documentation) 
```

Comment:
Suggest updating the title to "Fix or Improve Documentation" to be consistent with the first two titiles.